netmanager/src/views/components/Sites/SiteView.js

```diff
@@ -261,6 +264,32 @@ const SiteForm = ({ site }) => {
             required
           />
         </Grid>
+        <Grid items xs={12} sm={6} style={gridItemStyle}>
+          <TextField
+            id="search_name"
+            label="Editable Name"
+            defaultValue={site.search_name}
+            variant="outlined"
+            onChange={handleSiteInfoChange}
+            error={!!errors.search_name}
+            helperText={errors.search_name}
+            fullWidth
+            required
+          />
+        </Grid>
+        <Grid items xs={12} sm={6} style={gridItemStyle}>
+          <TextField
+            id="location_name"
+            label="Editable Description"
```

Comment:
Thanks @Codebmk 

1. Could we explicitly call them Mobile App names, etc?  Or something that is easily identifiable? Since they are mostly being used in the Mobile App?
2. Secondly, will these changes automatically be reflected in then mobile app? We need to also confirm this since I am aware that some sections of our systems fetch data from Big Query yet these changes are just within the device-registry microservice.

cc: @NoahNsimbe

Reply:
@Baalmart or @NoahNsimbe do you have some suggestions on names that are more identifiable?? The mobile app picks these values from the search_name and location_name fields

When the field is modified, this should automatically reflect in the mobile app. I've been informed that those are the fields used in the mobile app

Reply:
Hi @Codebmk , could we perhaps create a separate section for Mobile App site fields?
Just to make it easy for User to find them --- one the same page?
About the names, I am not sure but having separate sections can help do properly "distinguish them.

Reply:
Done @Baalmart 
![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/cab94380-ca51-45e5-aebf-cc578d828681)

Reply:
Thanks @Codebmk